guides/doc-Planning_for_Project/topics/Deployment_Considerations.adoc

```diff
@@ -192,17 +192,19 @@ Note that kickstart repositories should not be added to content views, as they a
 === Content Deployment
 
 Content deployment is the management of errata and packages on content hosts.
-There are two methods for content deployment on {Project}; the default is *remote execution*, and the second is the deprecated *katello agent*.
+There are two methods for content deployment on {Project}; the default method is *remote execution*, and the second method is the deprecated *Katello agent*.
 
-* *Remote execution* - Remote execution via SSH transport allows the install, update, or removal of packages, the bootstrap of configuration management agents, and the trigger of Puppet runs.
-This is the preferred method for content deployment.
+* *Remote execution* - Remote execution allows installation, update, or removal of packages, the bootstrap of configuration management agents, and the trigger of Puppet runs.
+You can configure {Project} to perform remote execution either over SSH (push based) or over MQTT/HTTPS (pull based).
+While {ProjectServer} has remote execution enabled by default, it is disabled by default on {SmartProxyServer}s and content hosts, and has to be manually enabled.
 +
-While {ProjectServer} has remote execution enabled by default, it is disabled by default on {SmartProxyServer}s and content hosts and has to be manually enabled.
+This is the preferred method for content deployment.
```

Comment:
Personally, I would place this sentence right after the first sentence of the paragraph.

Reply:
I like it better in this place, because it stands out and it pairs logically with the note about Kattelo agent deprecation.